php-base/build-scripts/move-config-files.sh

```diff
@@ -47,15 +47,15 @@ fi
 
 # Move user-provided nginx config files.
 if [ -f "${NGINX_CONF_INCLUDE}" ]; then
-    mv "${NGINX_CONF_INCLUDE}" "${NGINX_USER_CONF_DIR}/nginx-app.conf"
```

Comment:
Strictly saying, I would separate this change from this PR because it's unrelated.